### ISSUE TITLE
Improve information hierarchy in tool listing

### ISSUE THREAD

**Issue by iadawn** [w3c/wai-authoring-tools#49]
When the details for an entry are revealed the accessibility features appear before the details of the tool. I think this should be reversed. In terms of information hierarchy it makes more sense that the summary description of the tool is followed immediately by the details (particularly website link) and then the accessibility features.

**Comment by hidde** [comment]
Thanks for this idea.

Details used to be above features, but I have since changed it, as I believe the accessibility features are more important than the “meta” data.

You're saying “particularly website link”, which I think is a great point… maybe that has been addressed sufficiently now that [we have made the title a link to the website](https://github.com/w3c/wai-authoring-tools/issues/48)?

Looking forward to hearing your thoughts, if you feel we should still move the features, I would propose we discuss it in EOWG to hear what the wider group thinks, I'm happy to leave the features where they are, or move them.

**Comment by slhenry** [comment]
@hidde In response to ideas discussed today, if we're going for tabs, I think they should be by status --
(**not** "Accessibility for content editors (editing experience)" and "Accessibility for end users (output)") -- 

although not sure the groupings -- maybe:
* Supported
* Partially
* Not Supported
* NA or Not Sure 
* About

but that's a lot of tabs :-(
so maybe:
* Supported
* Not or Partially (with: Partially, Not Supported, Not Applicable, Not Sure)
* About

**Comment by hidde** [comment]
Cool, agreed we should not have too many tabs… and I like your last suggestion (what would go into each tab). Not a huge fan of the last two names, but my English vocab is limited haha, so not sure what they should be… “About” is kind of “Meta info” (but that's much worseI think?). I'll stick with the names for now and trust on all of us to improve them if we can.

Removing from agenda and EOWG input wanted, because I now know what to do: create a proposal for tabs based on @slhenry's last feedback and report back here when that's done.

**Comment by hidde** [comment]
Proposal for tabs: https://5dd2a54fa36b450008a380e1--wai-authoring-tools.netlify.com/authoring-tools/ 

Features: 
- for optimal space usage, on small screens tabs are horizontal, on desktop they are vertical
- grouped in not too many tabs (as suggested by @slhenry) 

![](https://user-images.githubusercontent.com/178782/69096572-00320f00-0a55-11ea-9f5d-fb2feaf318df.png)
